contracts/TokenConverter/AbstractTokenConverter.sol

```diff
@@ -930,6 +964,11 @@ abstract contract AbstractTokenConverter is AccessControlledV8, IAbstractTokenCo
                     amountIn = amountToConvert;
                 }
 
+                bool isAmountInValid = _validateMinAmountToConvert(amountIn, tokenAddressOut);
+                if (!isAmountInValid) {
```

Comment:
Only a suggestion, feel free to ignore it :)

```suggestion
                if (!_validateMinAmountToConvert(amountIn, tokenAddressOut)) {
```

Reply:
Done https://github.com/VenusProtocol/protocol-reserve/pull/48/commits/554d804df86900e232a5055e1ca3c85a9f0d169e